region.go

```diff
@@ -30,6 +30,12 @@ func GetRegion(zone string) (region string, err error) {
 		region = "osa"
 	case strings.HasPrefix(zone, "mon"):
 		region = "mon"
+	case strings.HasPrefix(zone, "fra"):
+		region = "fra"
```

Comment:
I don't see any entries for this one in this [table](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-creating-power-virtual-server), is this still in staging or am I missing something here?

Reply:
In another channel, I saw `fra04` and `fra05` mentioned, but then noticed they actually used `eu-de-`.  Therefore I removed it.